test/test_logIn.py

```diff
@@ -0,0 +1,43 @@
+from django.test import LiveServerTestCase
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+
+
+class LogInTestCase(LiveServerTestCase):
+    def setUp(self):
+        driver = webdriver.Chrome(ChromeDriverManager().install())
+        driver.get("https://doogking-testing.azurewebsites.net/login")
```

Comment:
Hauríem de fer servir l'url https://doogking.azurewebsites.net en lloc de https://doogking-testing.azurewebsites.net, que el primer és on estem fent deploy amb el que sabem que funciona segur mentre que al de testing no s'assegura que tot sigui funcional ni actualitzat. Això és error meu, que no us ho he explicat en cap moment i els he anat utilitzant indistintament...

```suggestion
        driver.get("https://doogking.azurewebsites.net/login")
```

Reply:
Perfecte ho canvio.